src/tests/unit/Request.test.js

```diff
@@ -0,0 +1,19 @@
+const requestGenerator = require('../../Request')
+const chai = require('chai')
+const expect = chai.expect
+
+describe('Testing getter and setter in request', () => {
```

Comment:
I think the description of the tests should give a general description, and the "it" parts should have the structure 
"it(returns something when input is this and the status is that").